arwen/contexts/bigint.go

```diff
@@ -32,18 +30,12 @@ func (context *bigIntContext) PushState() {
 	context.stateStack = append(context.stateStack, newState)
```

Comment:
The new state has access to previous state's values / big integers, since `mapValues` isn't copied, but passed by reference. Is this correct & desired behavior?